public/js/gulliver.js

```diff
@@ -0,0 +1,95 @@
+/* eslint-env browser */
```

Comment:
Maybe don't need this quite yet, but need to use some sort of module system. (At the moment, everything is imported into global scope.)